content/en/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -16,13 +12,13 @@ Available options listed below have different tradeoffs. You should choose the o
 * [Workload Identity](#workload-identity) takes more efforts to set up, but allows fine-grained permission control. It is recommended for production use-cases.
 * [Google service account keys stored as Kubernetes secrets](#google-service-account-keys-stored-as-kubernetes-secrets) is the legacy approach and no longer recommended in GKE. However, it's the only option to use GCP APIs when your cluster is an [anthos](https://cloud.google.com/anthos) or on-prem cluster.
 
-{{% alert color="warning" %}}
-<p>NOTE: AI Platform Pipelines only supports Compute Engine default service account out of the box. If you want custom configurations, recommend using Pipelines Standalone instead. For details, please refer to <a href="https://cloud.google.com/ai-platform/pipelines/docs">AI Platform Pipelines documentation</a>.</p>
-{{% /alert %}}
-
 ## Before you begin
 [Installation Options for Kubeflow Pipelines](/docs/pipelines/installation/overview/) introduces options to install Pipelines. Be aware that authentication support and cluster setup instructions will vary depending on the option you installed Kubeflow Pipelines with.
 
+* For Kubeflow Pipelines standalone, you can compare and choose from all 3 options.
+* For full Kubeflow starting from Kubeflow 1.1, [Workload Identity](#workload-identity) is recommended and it's the only option configured by default.
+* For AI Platform Pipelines, [Compute Engine default service account](#compute-engine-default-service-account) is the only supported option.
```

Comment:
> it's the only option configured by default

This sounds a bit unclear to me. Is it the only supported option? or is it just the default option and users can choose others if they want?

Reply:
The docs describe options for more granular permissions. Would it be fair to say that it is the default option?

Reply:
It contains two parts:
1. using an option typically needs cluster setup, and we've done the setup for users with workload identity in Kubeflow.
2. It's the default and recommended option.

Reply:
So users can indeed choose others if they want, I'll rephrase as default option. Thank you!